src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, Color color) {
+        this.radius = radius;
+        super.color = color;
+    }
+
+    public void setRadius(int radius) {

```

Comment:
The setter 'setRadius' is not necessary if the 'radius' is set through the constructor and not supposed to change after the object is created. If the radius is meant to be mutable, keep the setter; otherwise, it can be removed.